Cargo.toml

```diff
@@ -70,13 +70,13 @@ async-trait = "0.1"
 backtrace = "0.3"
 batch-system = { path = "components/batch-system", default-features = false }
 byteorder = "1.2"
-case_macros = { path = "components/case_macros" }
-causal_ts = { path = "components/causal_ts" }
+case_macros = { path = "components/case_macros", default-features = false }
```

Comment:
Why the change?

Reply:
Some path dependencies use default features which eventually enables `test-engine-kv-rocksdb` and `test-engine-raft-raft-engine`. This makes it impossible to disable them and use other engines.

I didn't bother checking exactly which package is the root cause, instead I added an automated pass to force disable default features for all path dependencies.

Reply:
I think this is too verbose and confusing. For example, there is no features at all in `case_macros`. You can check what features are enabled by `cargo tree -e features`.

Reply:
We don't have CI pipelines to exhaustively check all custom feature combinations. Forcing them to disable at least avoids feature conflicts for arbitrary feature builds. As for you example, the `case_macros` might have features in the future, but we have no way to properly guard it except unconditionally disallowing using default features.

Reply:
> the case_macros might have features in the future,

`case_macros` has been created for more than 1 year without any changes.

Reply:
The point is the user package cannot deduce whether a dependency package have default features. Unless you have a way to assert a package have no default features, the only way to programmatically make sure the feature doesn't conflict is to disallowing the possibility entirely.